Core/VariantManager.swift

```diff
@@ -104,8 +108,11 @@ public class Arc4RandomUniformVariantRNG: VariantRNG {
 
     public init() { }
 
-    public func nextInt(upperBound: Int) -> Int {
-        return Int(arc4random_uniform(UInt32(upperBound)))
```

Comment:
Why change this? Was there a problem with using arc4random_uniform? I would expect it to be fine for our needs. Looking forward to Swift 4.2 new random methods soon!